src/pages/usage.md

```diff
@@ -52,27 +52,30 @@ xychart-beta
 In November 2022, we moved from email lists hosted by Google Groups to a Discourse forum. 
 
 As of March 20, 2024, over the last 12 months:
-* 244 new users signed up on our forum for a total of 470 users. 
-* 324 topics were created for a total of 1,700 messages.
+* 272 new users signed up[^1] on our forum for a total of 608 users[^2].
```

Comment:
Should the date above also be changed from March 20 to September 25?

Reply:
this should have been fixed via #382